bas-platform/src/actions/basctlServer.ts

```diff
@@ -12,7 +12,7 @@ function handleRequest(socket: net.Socket) {
     socket.on('data', dataBuffer => {
         const data: any = getRequestData(dataBuffer);
 
-        if (data.command === 'open') {
+        if (data.command === 'openBrowser') {
             const uri = vscode.Uri.parse(data.url, true);
             vscode.env.openExternal(uri);

```

Comment:
this command can open any uri and not just browser (http:, https:) depending on the used scheme

Reply:
if (data.command === 'openExternal') {